### PR TITLE
Added cache control

### LINKED ISSUE #1072: Add cache control for the Splunk app
**Describe the solution you'd like**
Improve some strategies for cache control.

- [x] Add meta for cache-control in the head section of the page.
- [x] Verify the app version for backend vs browser version (then force update bundles).
- [x] Avoid infinite loop for the previous step. Then show a message to clear the cache manually.

### PR DESCRIPTION
**Pull request by Machi3mfl**
Hi Team, this PR add
- Show a warning toast after app loaded if the app version of browser and backend are differents. The message toast advices to the user that need to clear browser cache.
- Added a `prebuild` script that refresh the version and revision number.

Closes #1072

### CODE REVIEW

**Comment by Machi3mfl** [comment]
Screen Shot 2021-04-28 at 17 57 40

File: SplunkAppForWazuh/appserver/static/js/services/app-version/appVersionService.js [hunk 1]
```diff
@@ -13,7 +21,7 @@ define(['../module'], function (module) {
          * Set the info about the app and splunk
          */
         const setAppInfo = (info) => {
-            let appInfo = info;
```
Comment (CPAlejandro) [inline SplunkAppForWazuh/appserver/static/js/services/app-version/appVersionService.js:16]: Would it be missing here to declare the appInfo variable?

**Review by CPAlejandro — CHANGES_REQUESTED** [review]

**Review by Machi3mfl — COMMENTED** [review]
Comments revolved

**Comment by frankeros** [comment]
The script is changing the const name 
image
I'm fixing this now

**Review by pablomarga — APPROVED** [review]
LGTM!

**Pull request merged**